jerry-core/mem/mem-poolman.c

```diff
@@ -34,16 +34,21 @@
 #define MEM_ALLOCATOR_INTERNAL
 #include "mem-allocator-internal.h"
 
+ /**
+ * Type for mem_pool_chunk
+ */
+typedef struct mem_pool_chunk mem_pools_chunk_t;
+
 /**
  * Node for free chunk list
  */
-typedef struct
+struct mem_pool_chunk
 {
-  struct mem_pools_chunk_t *next_p; /* pointer to next pool chunk */
+  mem_pools_chunk_t *next_p; /* pointer to next pool chunk */
 #ifndef MEM_HEAP_PTR_64
   uint32_t dummy; /* dummy member for alignment */
 #endif
-} mem_pools_chunk_t;
+} mem_pool_chunk;
```

Comment:
Yes, you are right. I changed this with another solution.
This modification fix the following warnings:
ezeket fixálja:
150:22: assignment from incompatible pointer type : mem_free_chunk_p = chunk_p->next_p;
173:27: assignment from incompatible pointer type : chunk_to_free_p->next_p = mem_free_chunk_p;
190:39: initialization from incompatible pointer type : mem_pools_chunk_t *const next_p = mem_free_chunk_p->next_p;